js/data/buffer.js

```diff
@@ -267,20 +280,4 @@ function align(value, alignment) {
     return value;
 }
 
-function checkNumber(input) {
-    if (!isNaN(parseFloat(input)) && isFinite(input)) {
-        return input;
-    } else {
-        throw new Error('Validation failure in buffer push method creation');
-    }
-}
-
-function checkAttributeTypeName(input) {
-    if (Object.keys(Buffer.AttributeType).indexOf(input) === -1) {
-        throw new Error('Validation failure in buffer push method creation');
-    } else {
-        return input;
-    }
-}
```

Comment:
@tmcw @jfirebaugh Is this change in validation for the `eval`ed code ok? (These checks are now done in the constructor). I'm not sure where the standards are.